bndtools.core/bndtools.shared.bndrun

```diff
@@ -251,30 +251,30 @@ crossplatform.runprops: \
 	org.w3c.dom.events;version='[3.0.0,3.0.1)',\
 	org.w3c.dom.smil;version='[1.0.1,1.0.2)',\
 	org.w3c.dom.svg;version='[1.1.0,1.1.1)',\
-	biz.aQute.bnd.maven;version='[4.4.0,4.4.1)',\
-	biz.aQute.bndlib;version='[4.4.0,4.4.1)',\
-	biz.aQute.repository;version='[4.4.0,4.4.1)',\
-	biz.aQute.resolve;version='[4.4.0,4.4.1)',\
-	bndtools.api;version='[4.4.0,4.4.1)',\
-	bndtools.builder;version='[4.4.0,4.4.1)',\
-	bndtools.core;version='[4.4.0,4.4.1)',\
-	bndtools.jareditor;version='[4.4.0,4.4.1)',\
-	bndtools.m2e;version='[4.4.0,4.4.1)',\
-	bndtools.m2e.debug.fragment;version='[4.4.0,4.4.1)',\
-	bndtools.pde;version='[4.4.0,4.4.1)',\
-	bndtools.release;version='[4.4.0,4.4.1)',\
+	biz.aQute.bnd.maven;version='[5.0.0,5.0.1)',\
```

Comment:
You will need to remove this file from the PR as Peter changed it already in a PR he has merged.

Reply:
Done.